cow_amm/uni_v2_amm_lp_and_tvl_4047194

```diff
@@ -0,0 +1,94 @@
+WITH 
+cow_amm_pool as (
+select blockchain, address, token_1_address, token_2_address
+from query_3959044 q
+where address={{cow_amm_pool}}
```

Comment:
I wonder if this is the right input for the comparison? This way they can only be used for CoW AMM pools (where in theory profitability could be computed for any AMM regardless whether there exists a CoW AMM)\

Maybe we should have all queries be parametered on token1, token2 and then put the search for CoW AMMs in the top query that calls the subqueries.

Reply:
I thought of adding token_1 and token_2 parameters, the issue see here is how to know which token is 1 and which one is 2? I think there is an alphabetical order but does it always stand? Should we allow token symbol inputs or raw tokens (more accurate)?

Reply:
Def. addresses and not symbols. As to ordering, if we cannot make it ordering agnostic you can also follow @fhenneke's suggestion of using the uniswap pool as an argument. Note, that this query is not designed to be used by itself but instead will be embedded in a simple visualisation query which can make sure the input parameter is well defined (e.g. by choosing it from a hard coded list or the like).

Reply:
Thinking about this more, I believe the input should be two tokens. This way each "$10k growth" chart is independent of cow amms (e.g. we could still compare uni, balancer and a counterfactual rebalancing portfolio even if no CoW AMM existed).